core/templates/dev/head/pages/creator_dashboard/create_activity_modal_directive.html

```diff
@@ -15,15 +15,13 @@ <h1 translate="I18N_CREATE_ACTIVITY_TITLE"></h1>
           </button>
         </div>
 
-        {% if can_create_collections %}
-          <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 text-center">
+          <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 text-center" ng-if="can_create_collections">
```

Comment:
The indentation needs to be fixed; this and the following lines should be de-indented by 2.